Zend/tests/this_in_foreach_001.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+$this in foreach
+--FILE--
+<?php
+$a = [1];
+foreach ($a as $this) {
+	var_dump($this);
+}
+?>
+--EXPECTF--
+Fatal error: Cannot re-assign $this in %sthis_in_foreach_001.php on line 3
```

Comment:
This message is a bit strange. Since there is no `$this` in current scope, `re-assign` in the message doesn't make sense.